.taskcluster.yml

```diff
@@ -62,3 +62,79 @@ tasks:
           description: Building reference browser - triggered by a pull request.
           owner: ${event.pull_request.user.login}@users.noreply.github.com
           source: ${event.repository.url}
+  - $if: 'tasks_for == "cron"'
+    then:
+      $let:
+        decision_task_id: {$eval: as_slugid('decision_task')}
+        expires_in: {$fromNow: '1 year'}
+        repository: ${event.repository.clone_url}
+        scheduler_id: focus-nightly-sched
+      in:
+        taskId: ${decision_task_id}
+        taskGroupId: ${decision_task_id}   # Must be explicit because of Chain of Trust
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '2 hours'}
+        expires: ${expires_in}
+        schedulerId: ${scheduler_id}   # Must be explicit because of Chain of Trust
+        provisionerId: aws-provisioner-v1
+        workerType: gecko-focus   # This workerType has ChainOfTrust enabled
+        requires: all-completed   # Must be explicit because of Chain of Trust
+        priority: medium
+        retries: 5
+        scopes:
+          - queue:scheduler-id:${scheduler_id}
+          - queue:create-task:highest:aws-provisioner-v1/gecko-focus
+          - queue:create-task:highest:scriptworker-prov-v1/mobile-signing${scriptworker_environment}-v1
+          - queue:create-task:highest:scriptworker-prov-v1/mobile-pushapk${scriptworker_environment}-v1
+          - project:mobile:reference-browser:releng:signing:cert:${signing_environment}
+          - project:mobile:reference-browser:releng:signing:format:autograph_apk_reference_browser
+          - project:mobile:focus:releng:googleplay:product:reference-browser${pushapk_environment}
```

Comment:
I'm not sure how I feel about this string interpolation, but I think it's better than passing in entire scopes via variables, because then we'll have half our scopes in `schedule_nightly_graph.py`, and the other half here in the `.taskcluster.yml` file.
Another option is just to pass an `is_staging` variable and use `$if`, but I'd rather not have imperative logic in the template (:thinking:)